lib/resources/majors.tsx

```diff
@@ -2711,6 +2711,308 @@ const bsMolCell: Major = {
+          required_credits: 9,
+          description:
+            '<b>Computational neuroscience (CP)</b> <br /> ' +
+            'EN.553.291 (or AS.110.201 and AS.110.302) is required in addition to the credits required of the focus area.',
+          criteria: 'NEUR-CP[T]',
+        },
+        {
+          required_credits: 9,
+          description:
+            '<b>Cellular and molecular neuroscience (CM)</b> <br /> ' +
+            'AS.020.306 and AS.020.316 must be selected as the required biology course with lab.',
+          criteria: 'NEUR-CM[T]',
+        },
+      ],
+    },
+    {
+      name: 'Elective Course',
+      required_credits: 3,
+      min_credits_per_course: 3,
+      description:
+        'Three credits of 300-level or higher approved course outside of selected focus area',
+      criteria: 'AS.080.3[C]',
+    },
+    {
+      name: 'Research',
+      required_credits: 6,
+      min_credits_per_course: 0.5,
+      description:
+        'Six credits of neuroscience research, obtained through work in one of the neuroscience laboratories ' +
+        'participating in the program. Read more about <a href="https://krieger.jhu.edu/neuroscience/research/research-credit-requirements/">' +
+        'research credit requirements</a>.',
+      criteria: 'AS.080.500[C]^OR^AS.080.541[C]', // TODO: course # for neuro research?
+      fine_requirements: [
+        {
+          required_credits: 0.5,
+          description:
+            '<b>Scientific Communication</b> <br /> Read more about <a href="https://krieger.jhu.edu/neuroscience/research/research-credit-requirements/">' +
+            'research credit requirements</a>.',
+          criteria: 'AS.080.500[C]',
+        },
+      ],
+    },
+    {
+      name: 'Humanities (H) Distribution',
+      required_credits: 9,
+      min_credits_per_course: 3,
+      description:
+        'The distribution requirement stipulates that students must earn a minimum number of credits in academic areas outside of their primary major. ' +
+        'The student must complete at least 9 credits in the Humanities (H) area. ' +
+        'These credits fulfilling the distribution requirement may overlap with major or minor requirements and the writing-intensive requirement.',
+      criteria: 'H[A]',
+    },
+    {
+      name: 'Social Science (S) Distribution',
+      required_credits: 9,
+      min_credits_per_course: 3,
+      description:
+        'The distribution requirement stipulates that students must earn a minimum number of credits in academic areas outside of their primary major. ' +
+        'The student must complete at least 9 credits in the Social Science (S) area. ' +
+        'These credits fulfilling the distribution requirement may overlap with major or minor requirements and the writing-intensive requirement.',
+      criteria: 'S[A]',
+    },
+    {
+      name: 'Writing Intensive',
+      required_credits: 12,
+      min_credits_per_course: 3,
+      double_count: true,
```

Comment:
`double_count` not required, but confirm that you dont need it before you delete it.